modules/Data_Cleaning/lab/Data_Cleaning_Lab_Key.Rmd

```diff
@@ -148,13 +148,15 @@ BloodType %>% count(type)
 
 6. Make a new tibble of `BloodType` called `Bloodtype_split` that splits the `type` variable into two called `blood_type` and  `Rhfactor`. Note: periods are special characters that generally are interpreted as wild cards thus we need `\\` to tell R that we want it to be interpreted as a period.
 
-The following commented code within the chunk (starts with `#`) can be used as a guide to help you. Type your code below this commented code. 
+```
+# General format
+NEW_TIBBLE <- OLD_TIBBLE 
+    %>% separate(OLD_COLUMN, into = c("NEW_COLUMN1", "NEW_COLUMN2", ...), sep = "SEPARATING_CHARACTER")
```

Comment:
```suggestion
    %>% separate(OLD_COLUMN, into = c("NEW_COLUMN1", "NEW_COLUMN2"), sep = ""\\.")
```

Reply:
I thought the `...` might confuse students into thinking that they needed more columns... also with the "separating_character" since we don't show this otherwise, I thought it might be worth writing it out

Reply:
What if they want to use something other than "." ?

Reply:
I guess I was thinking this is more "general" format (they might need more columns in general use case)

Did you want this to be specific to the question for both of these?

Reply:
I see your point, I just think the leap of reading the directions to implementing it the first time might through them for a loop. Maybe we can just add anote saying recall that you can separate by other separators?

Reply:
or maybe we can say in the directions that the  "separating character" will be "\\."?

Reply:
I am fine with you just stating such :)

Reply:
I think that covers both of our concerns hopefully